rest-api-spec/src/main/resources/rest-api-spec/test/cat.nodes/10_basic.yml

```diff
@@ -58,6 +58,19 @@
       $body: |
                /^      http \n ((\d{1,3}\.){3}\d{1,3}:\d{1,5}\n)+  $/
 
+  - do:
+      cat.nodes:
+          h: disk,diskAvail,diskTotal,diskUsed,diskUsedPercent
```

Comment:
That is a pretty good idea @javanna, I removed the alias for `diskAvail` and added another test for the aliases. Should I fix up the commit or add another commit on top of this one?